tasks/akeyless/src/main/java/com/walmartlabs/concord/plugins/akeyless/SecretExporter.java

```diff
@@ -0,0 +1,25 @@
+package com.walmartlabs.concord.plugins.akeyless;
+
+/*-
+ * *****
+ * Concord
+ * -----
+ * Copyright (C) 2017 - 2022 Walmart Inc., Concord Authors
+ * -----
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =====
+ */
+
+public interface SecretExporter {
+    String exportAsString(String o, String n, String p) throws Exception;
```

Comment:
Nit: rename parameters to `org`, `secretName` and `password`, perhaps?

Reply:
yep yep yep